### ISSUE TITLE
Web Client

### ISSUE THREAD

**Issue by brentkirkland** [UCSB-CS56-Projects/cs56-utilities-GEscraper#18]
Make a web client that runs off your csil account. May want to consider ruby on rails, node, django, or flask. Or spring if you LOVEEEE java.  

Save past searches and create an account on this web app. 

~500 pts

**Comment by PazZaitGivon** [comment]
This is a pretty large undertaking for a legacy code issue but for purpouses of W15 lab08 it is fine

**Comment by Simon-Huynh** [comment]
Convert the scraper into a webapp through the use of Heroku. (referencing W16 lab 08)  

~500

**Comment by EasterAndJay** [comment]
W16 approved

**Comment by SElegans** [comment]
F16 OK (500 pts)

**Comment by scottpchow23** [comment]
Yep this is difficult. Here's a [link](https://devcenter.heroku.com/articles/deploying-java) to help get you started.

500 xp

W18 OK
